src/fidesops/ops/api/v1/endpoints/connection_endpoints.py

```diff
@@ -356,3 +370,25 @@ async def test_connection_config_secrets(
     connection_config = get_connection_config_or_error(db, connection_key)
     msg = f"Test completed for ConnectionConfig with key: {connection_key}."
     return connection_status(connection_config, msg, db)
+
+
+def requeue_requires_input_requests(db: Session) -> None:
+    """
+    Queue privacy requests with request status "requires_input" if they are no longer blocked by
+    access manual webhooks.
```

Comment:
It'd be better to say explicitly this is if all access manual webhooks are disabled or deleted.

Reply:
Thanks Sean, I've tried to improve this docstring